### ISSUE TITLE
validation accuracy

### ISSUE THREAD

**Issue by roshanshaw3117** [SaiHitesh16/Glaucoma-Detection-using-CNN#2]
bro validation accuracy not coming as expected is there any solution of it?

**Comment by SaiHitesh16** [comment]
are u trying on the same datasets?

**Comment by roshanshaw3117** [comment]
yes validation accuracy is around 70%

**Comment by SaiHitesh16** [comment]
Can you provide the screenshot of accuracy graph or epochs  so that I may help you

**Comment by roshanshaw3117** [comment]
Epoch 148/150
32/32 [==============================] - 32s 914ms/step - loss: 0.0019 - accuracy: 1.0000 - val_loss: 1.1507 - val_accuracy: 0.6797
Epoch 149/150
32/32 [==============================] - 31s 991ms/step - loss: 8.3529e-04 - accuracy: 1.0000 - val_loss: 0.9458 - val_accuracy: 0.6562
Epoch 150/150
32/32 [==============================] - 32s 1s/step - loss: 0.0020 - accuracy: 1.0000 - val_loss: 1.0846 - val_accuracy: 0.6797

**Comment by SaiHitesh16** [comment]
The model was overfitted. How many images are you using in total and for training,validation.

**Comment by roshanshaw3117** [comment]
same as yours do i need to ensemble

**Comment by SaiHitesh16** [comment]
I have not used it. Try to re run the epochs code and check whether there is any change and also try to decrease the epochs value because training accuracy is 1.00.

 what is test accuracy ?

**Comment by roshanshaw3117** [comment]
65%

**Comment by SaiHitesh16** [comment]
ok try changing split ratio and check or can you provide the link for your code so that I can understand the issue better.

**Comment by roshanshaw3117** [comment]
ok

**Comment by Vishal321kumar** [comment]
can u pls help me with the project 
![Screenshot (102)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/0ac35a9d-5abe-4c20-b226-d64f9f7bf664)
![Screenshot (104)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/c2c860fa-9561-4657-88af-9d325141dde9)
?

**Comment by Vishal321kumar** [comment]
#1 model overfitting but is is actually same as urs
# low accuracy and wrong output 
![Screenshot (102)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/75215c29-9c30-454b-8808-c2ee1711e11d)
![Screenshot (104)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/30afdc85-3948-4f63-b998-1b861b83ff39)
